### ISSUE TITLE
feat: Hide 'Subtitle/CC turned on/off' popup

### ISSUE THREAD

**Issue by RamPawar7** [revanced/revanced-patches#2374]
### Application

Youtube (com.google.android.youtube)

### Issue

When watching a video in full screen mode, the popup 'Subtitles/CC turned on' that appears on the bottom left corner when enabling the Subtitles/CC is overlayed on top of the subtitles and hides them behind it for about 4 seconds.

### Patch

The patch will have option to either disable the 'Subtitles/CC turned off/on' popup completely or reduce the time for which the popup stays on screen. This can also be applied to other similar popups.

### Motivation

Many people that usually keep the subtitles turned off, whenever they don't understand a word or a sentence said in a video, they turn on subtitles briefly to read what was said. But they need to wait for this annoying popup to disappear and then they can rewind the video to read the subtitles.

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by Aunali321** [comment]
@LisoUseInAIKyrios no, it's only needed to add `return-void` instruction at first index. I PR'ed it [before](https://github.com/revanced/revanced-patches/pull/1499) but i don't remember the reason to close it (i think i had trouble adding settings). The patch works, i used it just 2 weeks ago albeit by manually changing smali instead of patcher. This is easy 2 minutes patch.

**Comment by Aunali321** [comment]
>  @Aunali321 is that to turn off the snackbar for all usage, or only to turn off the snackbar when CC is enabled?

> The same snackbar is used for many purposes, the first that comes to mind is the "feedback shared with creator" that is shown when disliking a video.

It seems to only disable toast for subtitles related things. i still get toast for disliking videos. LMK if you want to find the method.
